packages/accounts-controller/src/AccountsController.test.ts

```diff
@@ -1698,49 +1851,72 @@ describe('AccountsController', () => {
     });
   });
 
-  describe('getSelectedAccount', () => {
-    it('should return the selected account', () => {
-      const accountsController = setupAccountsController({
+  describe('setSelectedAccount', () => {
+    it('should set the selected account', () => {
```

Comment:
Lets not use the keyword `should` in our tests
```suggestion
    it('sets the selected account', () => {
```

https://github.com/MetaMask/contributor-docs/blob/main/docs/unit-testing.md#use-it-to-specify-the-desired-behavior-for-the-code-under-test